service/history/task/transfer_active_task_executor.go

```diff
@@ -606,28 +604,24 @@ func (t *transferActiveTaskExecutor) processStartChildExecution(
 		return nil
 	}
 
-	// Get parent domain name
-	var domain string
-	if domainEntry, err := t.shard.GetDomainCache().GetDomainByID(task.DomainID); err != nil {
+	// Get parent domainName name
```

Comment:
nit: no need to update comment

Reply:
Thanks for catching. Reverted the change.